### ISSUE TITLE
space between parens ()

### ISSUE THREAD

**Issue by schoening** [remimarsal/prettier-now-vscode#9]
https://github.com/prettier/prettier/issues/1303

**Comment by homestar9** [comment]
I agree @schoening. #1303 should be re-evaluated.  Many people follow the pattern of having spaces between parentheses.
